plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/onboarding-tasks.php

```diff
@@ -52,27 +50,26 @@ public function setUp(): void {
 		// Resetting task list options and lists.
 		update_option( Task::DISMISSED_OPTION, array() );
 		TaskLists::clear_lists();
-
 	}
 
 	/**
 	 * Tear down.
 	 */
 	public function tearDown(): void {
 		parent::tearDown();
-		$this->remove_color_or_logo_attribute_taxonomy();
+		$this->cleanup_test_product_attributes();
 		TaskLists::clear_lists();
 		TaskLists::init_default_lists();
 	}
 
 	/**
-	 * Remove product attributes that where created in previous tests.
+	 * Clean up product attribute taxonomies created during tests.
 	 */
-	public function remove_color_or_logo_attribute_taxonomy() {
+	public function cleanup_test_product_attributes() {
 		$taxonomies = get_taxonomies();
 		foreach ( (array) $taxonomies as $taxonomy ) {
-			// pa - product attribute.
-			if ( 'pa_color' === $taxonomy || 'pa_logo' === $taxonomy ) {
+			// pa_ prefix indicates product attribute taxonomy.
+			if ( in_array( $taxonomy, array( 'pa_color', 'pa_logo', 'pa_size' ), true ) ) {
```

Comment:
After removing `@runTestsInSeparateProcesses` annotation, the test failed with `Slug "size" is already in use. Change it, please.` error message. It seems that other tests are using the same slug, so I updated this method to clean up the `pa_size` taxonomy as well.